.github/workflows/release-docker.yml

```diff
@@ -10,24 +10,35 @@ jobs:
     name: Docker build and push
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-        with:
-          ref: master
       - name: Get version without v character
         id: version
         run: |
           VERSION=${{github.event.release.tag_name}}
           VERSION_WITHOUT_V=${VERSION:1}
-          echo "::set-output name=value::$(echo $VERSION_WITHOUT_V)"
-      - name: Release to Docker
-        run: | 
-          echo ${{secrets.DOCKER_PASSWORD}} | docker login -u ${{secrets.DOCKER_USERNAME}} --password-stdin
-          sleep 1m #docker image installs generator from npm, this sleep protects this step from any delays on npm side
```

Comment:
we need this delay still in the workflow. This workflow reacts on new GitHub Release, but npm publishing is still in the process and might be that this workflow starts then package is not published yet, so `npm install` installs not the latest version really.

Reply:
So maybe instead of delay we should couple them and prevent concurrency. Just waiting is a foul's game 😄

Reply:
what do you mean by `couple them`? the only option to not wait is to check with npm in loop if package is there - is it worth it?